File: sorting/js/selection-sort.js
let sort = [12,6,3,88,1,4,8]

for(let i=0; i<sort.length; i++){
    for(let j=0; j<sort.length; j++){
        // comparing array elements
        if(sort[j]>sort[i]){
            //swaping elements
            let temp = sort[i];
            sort[i]=sort[j];
            sort[j]= temp;
        }
    }
}

console.log(sort)

File: sorting/c-or-cpp/selection-sort.cpp
#include <iostream>
using namespace std;

int main(int argc, char const *argv[])
{   // length of array
	cout<<"Enter the length of array"<<endl;
	int n,temp;
	cin>>n;
	int arr[n];
	// Getting elements of array
	cout<<"Enter the elements of array"<<endl;
	for (int i = 0; i < n; i++)
	{
		cin>>arr[i];
	}
	for (int i = 0; i < n; i++)
	{
		for(int j = i; j<n; j++)
		{   // comparing elements of array
			if(arr[j]<arr[i])
			{
				temp = arr[i];
				arr[i]=arr[j];
				arr[j]=temp;
			}
		}
	}
	for(int i=0; i<n; i++)
	{
		cout<<arr[i]<<" ";
	}
	return 0;
}